ansible/roles/raf/cdap-perimeter-security/deploy/tasks/main.yml

```diff
@@ -0,0 +1,186 @@
+---
+# tasks file for cdap-parameter-security
+- name: Enabling security in CDAP (cdap-site)
+  shell: "/var/lib/ambari-server/resources/scripts/configs.py -a set -u {{ ambari_admin_user }} -p {{ ambari_admin_password }} -l {{ cdap_loadbalancer_vip }} -n {{ cluster_name }} -c {{ item.config }} -k {{ item.key }} -v {{ item.value }}"
```

Comment:
Please maintain state with ansible shell module using option "creates" for each property to make this idempotent